src/altinn-app-frontend/src/features/form/components/FullWidthWrapper.tsx

```diff
@@ -19,14 +20,25 @@ const useStyles = makeStyles({
       marginRight: '-96px',
     },
   },
+  consumeBottomPadding: {
+    marginBottom: '-24px',
+    '@media (min-width: 768px)': {
+      marginBottom: '-36px',
+    },
+  },
 });
 
-export function FullWidthWrapper({ children }: IFullWidthWrapperProps) {
+export function FullWidthWrapper({
+  children,
+  onBottom,
```

Comment:
This looks like an event - can we name it something else? 🙂 `isOnBottom` f.ex (or would it be `isAtBottom`? 🤔)

Should probably also have `false` as default value

Reply:
You're right! Renaming it and setting a default value.